modules/environment-roles/templates/shared_iam_terraform_policy.json.tpl

```diff
@@ -77,27 +72,22 @@
         "arn:aws:iam::${account_id}:role/TDRCreateDbUsersBastionRole${title(environment)}",
         "arn:aws:iam::${account_id}:role/TDRCreateDbUsersConsignmentapiRole${title(environment)}",
         "arn:aws:iam::${account_id}:role/TDRCreateKeycloak*",
-        "arn:aws:iam::${account_id}:role/TDRDbMigrationLambdaRole${title(environment)}",
+        "arn:aws:iam::${account_id}:role/*Lambda*${title(environment)}",
```

Comment:
Could we keep the `TDR`, just to control the access a bit more, eg: `.../TDR*Lambda*...`